docs(faq): update faq general (#1886)
scaleway/docs-content: faq/general.mdx

## Who is Scaleway?

> Scaleway is a global tech leader fulfilling our true mission of bringing you the cloud that makes sense. We offer the right combination of compute power and storage that is flexible, cost effective, reliable, secure and sustainably powered.

Author: ```suggestion
Scaleway is a global tech leader in cloud computing. We offer the right combination of computing power and storage that is flexible, cost-effective, reliable, secure, and sustainably powered.
```
